resources/views/components/torrent/row.blade.php

```diff
@@ -142,8 +142,6 @@ class="torrent-search--list__edit form__standard-icon-button"
                 </a>
             @endif
 
-            @livewire('small-bookmark-button', ['torrent' => $torrent, 'isBookmarked' => $torrent->bookmarks_exists, 'user' => auth()->user()], key('bookmark-torrent-'.$torrent->id))
-
             @if (config('torrent.download_check_page'))
```

Comment:
If bookmarks is still bugging out, there's basically a 100% chance that this new select code will bug out the exact same way.